multisrc/overrides/madara/norterosescan/src/NorteRoseScan.kt

```diff
@@ -0,0 +1,8 @@
+package eu.kanade.tachiyomi.extension.pt.norterosescan
+
+import eu.kanade.tachiyomi.multisrc.madara.Madara
+
+
+class NorteRoseScan : Madara("Norte Rose Scan", "https://norterose.com.br", "pt-BR"){
+
+}
```

Comment:
```suggestion
```

An empty overrides file is not necessary, the multisrc generator will do that for you.